cmd/system-probe/modules/network_tracer.go

```diff
@@ -300,6 +295,7 @@ func writeConnections(w http.ResponseWriter, marshaler encoding.Marshaler, cs *n
 
 	w.Header().Set("Content-type", marshaler.ContentType())
 	w.Write(buf) //nolint:errcheck
+	pool.Put(bs[:0])
```

Comment:
This means that each subsequent call to pool.Get() will return a len==0 slice, and protoSerializer will create a new slice. I think we want to return a slice of the original size.

Reply:
good point, will try this